Source/WebKit/Platform/SharedMemory.h

```diff
@@ -85,6 +92,8 @@ class SharedMemoryHandle {
 #endif
 
 private:
+    SharedMemoryHandle(Type&&, size_t);
+
```

Comment:
I think m_handle should be able to be non-mutable as of now..

Reply:
I wasn't able to remove `mutable` because of the generated messages.